### ISSUE TITLE
[Snackbar] Allow VoiceOver and Switch Control users to "exit" the Snackbar view

### ISSUE THREAD

**Issue by romoore** [material-components/material-components-ios#4798]
When a Snackbar Message appears, users of VoiceOver and Switch Control should be able to navigate into and out of the message view. When an action button is available and the focus shifts, users should not become "trapped" within the snackbar message.
